### ISSUE TITLE
XRPL Developer Discord

### ISSUE THREAD

**Issue by yubnubb** [XRPLF/xrpl.js#2774]
How do I get into the developer discord? Each time I try it continues to say that it is "unable to accept invite". Do I need a special invite? Currently, I am exploring NFT transactions on the XRPL and believe it would be beneficial to be a part of the discord dev community. Can anyone help me out? How do I get in?

**Comment by intelliot** [comment]
Which link are you using? https://xrpldevs.org?

**Comment by yubnubb** [comment]
Yes, Sorry for the late response. I used the above link and can't get in. Same error.

**Comment by yubnubb** [comment]
![Screenshot 2024-09-19 075948](https://github.com/user-attachments/assets/f9b74912-4cb8-41f5-9a19-df0e42c54bd7)

How do I get an invite that looks like this?

**Comment by yubnubb** [comment]
High there, yeah I have been using the above link. It doesn't seem to work.

Can you help me out? Trying to get involved.

Thanks,
Jan Kurt

**Comment by intelliot** [comment]
Here is the invite link that looks like that: https://discord.gg/xrpl

**Comment by yubnubb** [comment]
thanks for taking the time to help me out here. it also didn't work? Unable
to accept invite

**Comment by yubnubb** [comment]
my discord username is svennifty

**Comment by anissa-ripple** [comment]
@yubnubb are you still having issues joining the server?

**Comment by intelliot** [comment]
https://xrpldevs.org/ works for me. Closing due to lack of response to the last question above.
